tools/stronghold/src/api/compatibility.py

```diff
@@ -71,10 +72,25 @@ def check(
     before_api = api.ast.extract(before)
     after_api = api.ast.extract(after)
 
+    before_raw = api.ast.extract_raw(before)
```

Comment:
`extract` already calls `extract_raw` internally. 

Ideally instead of parsing AST two times we need to refactor `extract` and `extract_raw` to:
1. preserve the current semantics of the existing functions
2. have another function that returns both raw and mapped result reusing the implementation internally

alternatively, some use some kind of (short-term) memoization?